src/data/faq_de.json

```diff
@@ -28,6 +28,17 @@
                             "Als Medienvertreter richten Sie bitte Ihre Anfragen an <a href='mailto:press@sap.com'>press@sap.com</a>.",
                             "Die verschiedenen Feedback-Kanäle der <a href='https://www.coronawarn.app/de/community/'>Corona-Warn-App-Community<a/> helfen Ihnen bei Fragen zum Open Source Projekt der Corona-Warn-App."
                         ]
+                    },
+                    {
+                        "title": "So erreichen Sie uns aus dem Ausland - Internationale Telefonnummern",
+                        "anchor": "international_phone_numbers",
+                        "active": true,
+                        "textblock": [
+                            "Vom Ausland aus erreichen Sie uns unter folgenden internationalen Telefonnummern:",
+                            "Technische Hotline: +49 30 498 75401 erreichbar Montag bis Samstag von 7 bis 22 Uhr (außer an bundesweiten Feiertagen)",
+                            "TAN-Hotline: +49 30 498 75402 erreichbar 24 Stunden und sieben Tage die Woche",
+                            "Hinweis: Diese Rufnummern sind <b>nicht kostenfrei</b> - die Kosten für den internationalen Anruf richten sich nach dem lokalen Vertrag mit dem Provider des Anwenders."

```

Comment:
"mit dem Provider des Anwenders" -> "Ihrem Provider"

Reply:
auch hier - remove **lokal** ??

Reply:
yes "lokal" is unnecessary

Reply:
ok